src/config_options.rs

```diff
@@ -5,110 +5,104 @@
 // Full license information available in the project LICENSE file.
 //
 
-use crate::helper_functions::{should_start_service, strip_line_endings};
 use clap::Parser;
-use derive_getters::Getters;
-use log::debug;
-use std::{env, str};
+use log::LevelFilter;
 
-#[derive(Parser, Debug)]
-#[clap(name = "ACARS Router", author = "Mike Nye / Fred Clausen", version = "1.0", about = "ACARS Router: A Utility to ingest ACARS/VDLM2 from many sources, process, and feed out to many consumers.", long_about = None)]
-struct Args {
+#[derive(Parser, Debug, Clone)]
+#[clap(name = "ACARS Router", author, version, about, long_about = None)]
+pub struct Input {
     // Output Options
-    #[clap(short = 'v', long = "verbose", default_value = "info")]
     /// Set the log level. debug, trace, info are valid options.
-    verbose: String,
-    #[clap(long)]
+    #[clap(short, long, action = clap::ArgAction::Count)]
```

Comment:
Doing the verbosity levelling this way means that you can't easily use an environment variable, as it does a count of how many `v`s you have. By default it sets to error, then the following count applies:

`-v` -> Warn
`-vv` -> Info
`-vvv` -> Debug
`-vvvv` -> Trace

Reply:
If I'm understanding this right this breaks the compatibility with the python version of `acars_router`. It uses a `0/1/2` approach to indicate log level, and I made it so that it would work with existing installs with no change, but I did change it to be a more "readable" way in the documents by using `info/debug/trace` as the preferred flags.

I'm okay with breaking the functionality on this flag if @mikenye is.